runtime/Makefile

```diff
@@ -14,7 +14,8 @@ ENABLE_LIGHTNING?=ON
 ENABLE_LIGHTNING_KOKKOS?=OFF
 ENABLE_OPENQASM?=OFF
 ENABLE_ASAN?=OFF
-LIGHTNING_GIT_TAG_VALUE?="v0.34.0"
+ENABLE_LAPACK?=ON
+LIGHTNING_GIT_TAG_VALUE?="master"
```

Comment:
Can we target the minimum commit on the master branch that introduced support? Having a fixed hash will allow us to go back through Catalyst's git history and still build it successfully.